packages/lambda-functions/functions/import-language/index.ts

```diff
@@ -87,6 +87,14 @@ export const lambdaHandler = async (event: SQSEvent) => {
               state: GlossState.APPROVED,
             })),
           });
+          await client.glossHistoryEntry.createMany({
+            data: glossData.map(({ wordId, gloss }) => ({
```

Comment:
We are going to want to save the user who did the import so that we can retrace that if necessary. We'll probably need to add that to the import job in the database so that it is available in the lambda function

Reply:
Done.